test/metabase/sync/sync_metadata/sync_timezone_test.clj

```diff
@@ -0,0 +1,27 @@
+(ns metabase.sync.sync-metadata.sync-timezone-test
+  (:require  [expectations :refer :all]
+             [toucan.util.test :as tt]
+             [metabase.models
+              [database :refer [Database]]
+              [table :refer [Table]]]
+             [toucan.db :as db]
+             [metabase.sync.sync-metadata.sync-timezone :as sync-tz]
+             [metabase.test.data.datasets :as datasets]
+             [metabase.test.data :as data]
+             [clj-time.core :as time]
+             [metabase.util :as u]))
+
+(defn- db-timezone [db-or-id]
+  (db/select-one-field :timezone Database :id (u/get-id db-or-id)))
+
+(datasets/expect-with-engines #{:h2 :postgres}
```

Comment:
what exactly are we testing here? A quick comment would be nice since otherwise it's not really clear what's going on